src/frontend/src/pages/GanttPage/GanttPageWrapper.tsx

```diff
@@ -139,6 +142,7 @@ const GanttPageWrapper: FC = () => {
   const expandedHandler = (value: boolean) => {
     const ganttFilters: GanttFilters = { ...defaultGanttFilters, expanded: value };
     history.push(`${history.location.pathname + buildGanttSearchParams(ganttFilters)}`);
+    window.location.reload(); // quick fix for gantt chart not updating when any expander arrow is clicked
```

Comment:
Cool fix, hopefully people won't mind the page reloading. I think this works for expand/collapse but not for reset.

Reply:
Try the next commit